pipelines/A10/queries.log

```diff
@@ -43,3 +43,67 @@ SELECT T.offence_category, 100 * SUM(T.number_criminal_offences_solved) / SUM(T.
  Counterfeiting of Money, Official Stamps, Official Marks, Weights and Measures             22.13720
                                                                        Property             24.14896
                                                                         Liberty             45.04472
+
+Query Nr. 8
+-- How were the crime preventation rates in Switzerland for 2022 per crime category. Also provide the number of attempted and commited crimes? Order by prevention rate.
+SELECT T.offence_category, SUM(T.number_criminal_offences_attempted) AS number_of_attempted_crimes,   SUM(T.number_criminal_offences_registered) AS number_of_committed_crimes,   ROUND(100 * SUM(T.number_criminal_offences_attempted) / SUM(T.number_criminal_offences_registered), 2) AS prevention_rate FROM criminal_offences_registered_by_police AS T JOIN spatial_unit AS S on T.spatialunit_uid = S.spatialunit_uid WHERE S.name='Switzerland'   AND S.country = TRUE   AND T.year = 2022   AND T.number_criminal_offences_registered != 0   AND T.offence_criminal_code != 'Offence - total' GROUP BY T.offence_category ORDER BY prevention_rate DESC;
```

Comment:
**question:** does `committed` - `attempted` really equals _prevented_ ? Could it not be just a failed attempt?

Reply:
This is true, maybe I should call it something else then. I just thought that the prevention rate of crimes might be interesting and something to ask for. And there would no other way to answer this. Prevention could also include to make it harder to be successful with a crime. If you have a suggestion on how to improve this I am open to it. Otherwise I would leave it as such.